content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,118 @@
+As with every other release, we're regularly noting improvements and updates to keep you in touch with what is being done to ReactOS.
+In this release, improvements range from FreeLoader fixes, Shell features, kernel fixes, NetKVM VirtIO bringup, further work on the Xbox port and support for NEC PC-9800.
+
+Note that it took us over a year to get this release in shape and fix regressions.
+As such, ReactOS 0.4.14 does not contain the very latest developments we advertised in 2021 on our blog and on social media.
+Consider this a maintenance release, and stay tuned for what's coming next!
+
+{{< figure link="/img/project-news/reactos-0414-released/banner.png" src="/img/project-news/reactos-0414-released/banner.png" >}}
+
+## "Send To" feature and Shell improvements
+One of the main highlights of this release is the amount of improvements done to the Shell component, which makes up a vital part of the ReactOS user experience.
+Katayama Hirofumi MZ is the pioneer of the "Send To" implementation, a feature of the Shell that can be used to send files or directories to a certain location.
+
+In addition, "Open File Location" and "Open Command prompt here" were also implemented thanks to him.
+Apart from the aforementioned features, let's not forget the bug fixes that affected the operability of the Shell.
+Katayama Hirofumi MZ fixed the scroll selection and Mark Jansen improved some COM (Component Object Model) interfaces, so that certain drag and drop regressions could be fixed.
+The implementations of "Send To" and "Open File Location" are shown in the screenshots below:
+
+{{< figure link="/img/project-news/reactos-0414-released/shell.png" src="/img/project-news/reactos-0414-released/shell.png" >}}
+
+## NEC PC-9800 boot support
+NEC PC-9800 (or PC-98 in short) is a series of 16-bit and 32-bit Japanese computers developed and manufactured by NEC.
+As that type of hardware is based on x86 processors, it is a relatively easy porting target.
+Nevertheless, every port to another architecture opens up possibilities for finding and fixing bugs in the kernel modules of ReactOS.
+
+Dmitry Borisov, a new ReactOS contributor, helped to get the PC-98 port in shape.
+While there is stuff left to be done in the development of PC-98 boot support, Dmitry Borisov made great strides to get ReactOS booting on that platform.
+The following screenshot below demonstrates ReactOS booting on Neko Project 21/W emulator.
+
+{{< figure link="/img/project-news/reactos-0414-released/nec98.png" src="/img/project-news/reactos-0414-released/nec98.png" >}}
+
+## ICMP improvements
+Internet Control Message Protocol (or ICMP by short) is a protocol used by network device drivers to send network information such as errors or log status.
+The implementation was rather scarce and incomplete with pieces of code unimplemented or unfinished.
+The IP Helper module primarily uses this protocol and as a matter of that, it brought a lot of issues and bugs. 
```

Comment:
"The IP Helper module" --> what is that? a service? the iphlpapi.dll ? a driver?

Reply:
Likely iphlpapi.dll